ocs_ci/templates/ocp-deployment/inventory.yaml.j2

```diff
@@ -0,0 +1,11 @@
+[all:vars]
+ansible_user={{ ansible_user | default('test') }}
+ansible_become={{ ansible_become | default('True') }}
+
+openshift_kubeconfig_path={{ pod_kubeconfig }}
+openshift_pull_secret_path={{ pod_pull_secret | default('True') }}
```

Comment:
Is default value `True` correct for `..._path` variable?

Reply:
Done